core/src/main/scala/org/apache/spark/rdd/RDDCheckpointData.scala

```diff
@@ -71,6 +92,27 @@ private[spark] class RDDCheckpointData[T: ClassTag](@transient rdd: RDD[T])
     RDDCheckpointData.synchronized { cpFile }
   }
 
+  // Get the iterator used to write checkpoint data to HDFS
+  def getCheckpointIterator(
+    rddIterator: Iterator[T],
+    context: TaskContext,
+    partitionId: Int): Iterator[T] = {
```

Comment:
indent by 2 spaces, see the style guide: https://cwiki.apache.org/confluence/display/SPARK/Spark+Code+Style+Guide